javascript/fallback.ts

```diff
@@ -47,6 +47,28 @@ export function sqeuclidean(
   return result;
 }
 
+/**
+ * @brief Computes the L2 Euclidean distance between two vectors.
+ * @param {Float64Array|Float32Array|Int8Array} a - The first vector.
+ * @param {Float64Array|Float32Array|Int8Array} b - The second vector.
+ * @returns {number} The L2 euclidean distance between vectors a and b.
+ */
+export function euclidean(
+  a: Float64Array | Float32Array | Int8Array,
+  b: Float64Array | Float32Array | Int8Array
```

Comment:
[The 8-bit unsigned integer arrays are also supported](https://github.com/ashvardanian/SimSIMD/blob/c0807de559829e64ead0afb2563a279c4993d44f/include/simsimd/spatial.h#L59-L61).

Reply:
will add `Uint8` support as well. I guess I can also add Uint8 support for the squared euclidean fn as well ?

Reply:
If it's missing - sure ;)

Reply:
Unit8Array support was missing. Looks like the C API supports it, so I added it to `fallback.ts` as well !

Reply:
Uint8Array support was missing in `fallback.ts`. It seems to be supported with the C API, so I added support in the fallback js code as well !